Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:13
+
+WORKDIR /home
+
+COPY package*.json /home/
+
+RUN npm install
+
+COPY . /home
+
+EXPOSE 3000
```

Comment:
This is great 👍 
If you deploy this image where port 3000 is in use, the API will not be accessible and in some cases, the image will not start.
A workaround here is to have
```
ENV PORT 3000
EXPOSE $PORT
```

When running this and you need to use a different port, you will just need to `export PORT=8000` or add it to the `docker-compose.yaml`

Reply:
Will adjust in my next PR..